styles/questionpage.css

```diff
@@ -0,0 +1,21 @@
+.quizme-questionpages-container {
+  margin-top: 2rem;
+  padding: 2rem;
+  width: 45%;
```

Comment:
Not able to see the live preview so I am not sure but as per my understanding **width:45%** will shrink content to 45% of the viewport width even on smaller screens and it will not look good. Try to use **max-width** with rem or pixel unit as it will adjust according to viewport width.

Reply:
Yess , make sense , i will change that